### PR TITLE
fix example app iOS build; bump podspec version

### LINKED ISSUE #184: AppSettings.openNotificationSettings with app_settings: ^5.0.0
I get this error while trying to build the flutter application, anyone else dealing with the same? 

Error:: AppSettings.openNotificationSettings
app_settings: ^5.0.0
Flutter 3.7.11

### PR DESCRIPTION
**Pull request by navaronbracke**
This PR fixes a build issue in the example app on iOS due to a CocoaPods error.

It also updates the version references to `app_settings` and cleans up the changelog a bit.

Fixes #184

### CODE REVIEW

File: pubspec.lock [hunk 1]
```diff
@@ -185,5 +185,5 @@ packages:
     source: hosted
     version: "2.1.4"
 sdks:
-  dart: ">=2.19.0 <3.0.0"
-  flutter: ">=3.3.0"
+  dart: ">=3.0.0-0 <4.0.0"
```
Comment (navaronbracke) [inline pubspec.lock:188]: The current stable is 3.10.6, which includes Dart 3

File: example/ios/Runner.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -8,14 +8,13 @@
 
 /* Begin PBXBuildFile section */
```
Comment (navaronbracke) [inline example/ios/Runner.xcodeproj/project.pbxproj:9]: The changes here were made by XCode

**Comment by navaronbracke** [comment]
@spencerccf Can we look into landing this fix somewhere in the future?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,119 +1,135 @@
-## Platform Specifics
```
Comment (navaronbracke) [inline CHANGELOG.md:1]: This was no longer relevant

**Comment by spencerccf** [comment]
@ navaronbracke Please correct pubspec.yaml app version conflict... looks like it was changed down from 5.1.0 to 5.0.1 or something like that... needs to be greater than the master version based on the type of changes made... major.minor.patch

**Comment by navaronbracke** [comment]
@spencerccf Done, adjusted the version to become 5.1.1 instead

**Comment by gucci-quoci** [comment]
@spencerccf Any updates on this? Does it make sense to give @navaronbracke also write permissions so we can merge PRs faster and relieve you?

**Comment by gucci-quoci** [comment]
@spencerccf Can we merge and publish this fix?

**Comment by spencerccf** [comment]
Merged... will publish new version to pub.dev

**Pull request merged**